.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -3,3 +3,11 @@
 ### Changes proposed in this pull request
 
 ### Guidance to review
+
+### Checklist
+
+- [ ] Make sure all information from the Trello card is in here
```

Comment:
Are we saying cut-n-paste the Description here?

Reply:
I don't really have a preference.